draft-ietf-quic-transport.md

```diff
@@ -1719,9 +1719,11 @@ encrypted form in the token.
 A token issued with NEW_TOKEN MUST NOT include information that would allow
 values to be linked by an on-path observer to the connection on which it was
 issued, unless the values are encrypted.  For example, it cannot include the
-previous connection ID or addressing information.  Information that allows the
-server to distinguish between tokens from Retry and NEW_TOKEN MAY be accessible
-to entities other than the server.
+previous connection ID or addressing information.  Each NEW_TOKEN frame MUST
+be unique among all connections to that server, unless the frame is sent to
```

Comment:
Sorry for being pedantic, but this is a MUST and the requirement doesn't seem clear. First, this reads like a requirement on the client. If so,
- Does this imply that a client has to maintain and check _all_ NEW_TOKEN frames issued by every server for a long enough period of time?
- How would a client know whether a received NEW_TOKEN frame is a repair of a lost NEW_TOKEN frame?

If it's not meant to be a requirement on the client, how about "A server MUST ensure that every NEW_TOKEN frame it sends is unique across all clients, with the exception of those sent to repair loss of a previously sent NEW_TOKEN frame." 

Also, add a recommendation on what a client should do if it detects a server repeating NEW_TOKEN frames.

Reply:
(I'm ok with not having a client response to detecting dups ... I don't expect clients to be detecting this.)

Reply:
No, there is no obligation to check.  Do you think that we need to say as much?

The client should only see duplicates if there is a spurious loss detected by the server.

I like your suggested text and will use that.  The duplicate detection is already described on [line 5073](https://github.com/quicwg/base-drafts/pull/3281/files#diff-db016291106766877c4921a79f8596e0R5073).